docs/_includes/subscribe.html

```diff
@@ -11,27 +11,23 @@ <h3 class="section-subheading text-muted">{{ site.data.sitetext.subscribe.text |
       <div class="row">
         <div class="col-lg-6 text-lg-center" style="margin-left:auto;margin-right:auto;width:100%;">
           <form action="https://layer5.us15.list-manage.com/subscribe/post?u=6b50be5aea3dfe1fd4c041d80&amp;id=365c8c024d" 
-          method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" novalidate>
+          method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" onsubmit="document.getElementById('mc-embedded-subscribe-form').submit();">
             
-          <input type="text" value="" placeholder="First Name" name="FNAME" class="form-control" id="mce-FNAME" >
+          <input type="text" value="" placeholder="First Name" name="FNAME" class="form-control" id="mce-FNAME" required />
           <p class="help-block text-danger"></p>
-            <input type="text" value="" placeholder="Last Name" name="LNAME" class="form-control" id="mce-LNAME" >
+            <input type="text" value="" placeholder="Last Name" name="LNAME" class="form-control" id="mce-LNAME" required />
           <p class="help-block text-danger"></p>
           <input type="email" value="" name="EMAIL" class="form-control" id="mce-EMAIL" placeholder="Email Address" 
-          required style="white-space: nowrap;display: inline-block;">
+          required style="white-space: nowrap;display: inline-block;" required/>
```

Comment:
this `input` already has a required attribute.

Reply:
removed as requested. and pushed the changes